local/run.go

```diff
@@ -442,17 +458,29 @@ func (a *Agent) Stop() error {
 }
 
 func createAgentBootstrap(checkoutPath string) (*os.File, error) {
-	tmpFile, err := ioutil.TempFile(os.TempDir(), "bootstrap-")
+	tempFileNamePattern := "bootstrap-"
+	if runtime.GOOS == "windows" {
+		tempFileNamePattern = "bootstrap-*.bat"
```

Comment:
The `.bat` extension is really important, without it the agent won't find the boostrap script.